### ISSUE TITLE
No help topic for 'run'

### ISSUE THREAD

**Issue by drym3r** [opencontainers/runc#1875]
Hi

The readme [here](https://github.com/opencontainers/runc#running-containers), for example, says that is necessary to run `runc run container` to execute a container, but it returns `No help topic for 'run'`. It seems that the README is outdated, since `runc start` does what is expected about `runc run`.

**Comment by cyphar** [comment]
It looks like you're probably using an out-dated version of `runc`. Very old `runc` versions didn't have a `runc run` (and that feature was called `runc start`) -- but newer versions have both.

**Comment by drym3r** [comment]
I do not have an old version, I believe. I compiled the binary from the v0.1.1 git tag. When executing `runc -v` it returns `runc version spec: 1.0.0`. 
Fidgeting a little, I found that it depends if I execute runc run with or without root permission. I'm not sure if it's the expected behaviour, I still have a lot to learn:
```bash
$ runc run                                                                               
Incorrect Usage.
$ sudo runc run
No help topic for 'run'
```

**Comment by cyphar** [comment]
`v0.1.1` (from 2016) is much much older than `v1.0.0-rc5` (which is the latest release from a few months ago).

**Comment by drym3r** [comment]
Oh, sorry then. The last "stable" version is from 2016? Is there any problem on the development?

**Comment by cyphar** [comment]
The short answer to the question is "No, but we really shouldn't have done an 1.0-rc1 as early as we did -- the latest -rc release is still stable." Since we're an OCI project there are certain rules for post-1.0 projects and we were very cautious about actually stepping over it *but* a while ago we released 1.0.0-rc1 and we can't really undo that.

At the moment the -rcX is basically meaningless since those releases are clearly stable (they're used to run all containers under Docker, cri-o and family) and also added massive features (rootless containers were added in -rc4). It's just that the version string has some rules associated with it that make our lives difficult when we release 1.0.

With that said, the last release (1.0.0-rc5) was agreed to be "the last -rc". While I am the defacto release manager, I'm not sure how the other maintainers feel about the 1.0 question nowadays.
